ports/libusb/portfile.cmake

```
@@ -41,9 +46,15 @@ if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
       vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig/libusb-1.0.pc" " -lusb-1.0" " -llibusb-1.0")
   endif()
 else()
+    if(${DISABLE_UDEV})
```

The usual pattern is:
```suggestion
    if("feature-name" IN_LIST FEATURES)
```

---

Fixed in latest commits